goodreads-predictor/src/goodreads_predictor/pipelines/feature_engineering/pipeline.py

```diff
@@ -5,9 +5,13 @@
 
 from kedro.pipeline import Pipeline, pipeline, node
 from .nodes import apply_book_attributes
```

Comment:
Suggestion: Consolidate to one line

`from .nodes import apply_book_attributes, merge_description_embeddings, perform_clustering_analysis`